BridgeCareApp/BridgeCareCodeFirst/Models/DeficientData.cs

```diff
@@ -0,0 +1,125 @@
+﻿using BridgeCareCodeFirst.Models;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using System.Linq;
+
+namespace BridgeCare.Models
+{
+    public class DeficientData
+    {
+        public DeficientOrTargetTab GetDeficient(SimulationResult data, int[] totalYears)
+        {
+            var db = new BridgeCareContext();
+            IQueryable<DeficientResult> deficientOrTargetList = null;
```

Comment:
Naming.

I already know its an List or enumerable. Tell me about its purpose. How is it used? Just deficients or targets is fine here.